### DISCUSSION TITLE
Will server-side injection of ads affect FreeTube?

### DISCUSSION THREAD

**Discussion by taivlam** [FreeTubeApp/FreeTube#5364] [Q&A]
According to this [blog](https://tuta.com/blog/youtube-adblock-attack-continues) post from Tuta, YouTube is experimenting (sounds like [A/B testing](https://en.wikipedia.org/wiki/A/B_testing)) of server-side injected ads.

Will this affect FreeTube?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
See https://github.com/FreeTubeApp/FreeTube/discussions/5313
